src/main/scala/beam/router/skim/core/AbstractSkimmer.scala

```diff
@@ -65,10 +65,24 @@ abstract class AbstractSkimmerReadOnly extends LazyLogging {
   private[core] var currentIterationInternal: Int = -1
   private[core] var aggregatedFromPastSkimsInternal = Map.empty[AbstractSkimmerKey, AbstractSkimmerInternal]
   private[core] val pastSkimsInternal = mutable.HashMap.empty[Int, Map[AbstractSkimmerKey, AbstractSkimmerInternal]]
-
+  var numberOfRequests: Long = 0
+  var numberOfSkimValueFound: Long = 0
+  var numberOfDefaultSkimValueFound: Long = 0
```

Comment:
I do not see where 'numberOfDefaultSkimValueFound' is calculated, it seems it will always be 0.
Giving it a second thought - let's remove the `numberOfDefaultSkimValueFound` statistic at all, two of the rest is enough.